lib/remap-functions/src/parse_url.rs

```diff
@@ -0,0 +1,178 @@
+use remap::prelude::*;
```

Comment:
This file got moved to `lib/vrl/stdlib/src/parse_url.rs`

Reply:
👍 I think this snuck back in in a bad merge. I'll remove `lib/remap*`.